src/frontend/screens/Login/components/SIDLogin/index.css

```diff
@@ -54,6 +54,18 @@
   color: var(--link-highlight);
 }
 
+.loginInstructions .login-link {
+  display: flex;
+  background-color: var(--background-darker);
+  border-color: var(--input-background);
```

Comment:
To make this buttons align with the themes i think it should be `--accent`. The icon and text color should also be `--accent`.

Reply:
Updated with accent color